configs/sites/acorn/compilers.yaml

```diff
@@ -14,9 +14,27 @@ compilers:
     - intel/19.1.3.304
     - craype/2.7.13
     environment:
+      prepend_path:
+        PATH: '/opt/cray/pe/gcc/11.2.0/snos/bin'
```

Comment:
Have you verified that this works with your Python installation? Which Python are you using?

The reason I am asking is that when we use our own Miniconda Python 3.9, we need to use gcc@9 or gcc@10 to avoid `glibc` library conflicts.

Reply:
See the discussion here: https://github.com/NOAA-EMC/spack-stack/discussions/99

Reply:
I tested it on acorn. But did not try miniconda python 3.9. I will do one more test on it. We can use gcc/10 if 11 doesn't work.